hack/verify-swagger-spec.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+cd "${OS_ROOT}"
+
+SPECROOT="${OS_ROOT}/api/swagger-spec"
+TMP_SPECROOT="${OS_ROOT}/_tmp/swagger-spec"
+_tmp="${OS_ROOT}/_tmp"
+
+mkdir -p "${_tmp}"
+cp -a "${SPECROOT}" "${TMP_SPECROOT}"
+
+echo "generating fresh swagger spec..."
+"${OS_ROOT}/hack/update-swagger-spec.sh" &>/dev/null
```

Comment:
Don't clobber files I care about.  Make the update accept an optional target directory and pass an actual tmp location.  I know the other verifies clobber something in my source directory, but I don't like that either.